src/controllers/comment.controller.js

```diff
@@ -13,14 +16,26 @@ class CommentController {
   static async addCommentOnTripRequest(req, res) {
     const userId = req.signInUser.id;
     const { tripId, comment } = { ...req.body, ...req.params };
+    const { dataValues } = await TripService.findTripByProperty({ id: req.params.tripId });
+    const requester = await UserService.findUserByProperty({ id: dataValues.userId });
 
     const commentInformation = {
       userId,
       subjectId: parseInt(tripId, 10),
       subjectType: 'Trip',
       comment
     };
-    await CommentService.createComment(commentInformation);
+    const createdComment = await CommentService.createComment(commentInformation);
+
+    const lineManager = await UserService.findUserByProperty({ id: req.userData.lineManagerId });
```

Comment:
@erickyvand You can get the lineManager as well when retrieving the user role in the middleware.